plottr/__init__.py

```diff
@@ -1,8 +1,13 @@
 import os
 
-from pyqtgraph.Qt import QtCore, QtGui, QtWidgets
-Signal = QtCore.pyqtSignal
-Slot = QtCore.pyqtSlot
+from pyqtgraph.Qt import QtCore, QtGui, QtWidgets, QT_LIB
+
+if QT_LIB == 'PyQt5':
+    Signal = QtCore.pyqtSignal
+    Slot = QtCore.pyqtSlot
+elif QT_LIB == 'PySide2':
+    Signal = QtCore.Signal
+    Slot = QtCore.Slot
```

Comment:
do we want a meaninful `else:` here as well?

Reply:
We shold probably raise just to be safe